dev-scripts/prepare_pecl_release.py

```diff
@@ -49,12 +49,12 @@ def run(args):
 
 # replace the version number in the ext/crt.c file
 data = ""
-with open("ext/crt.c", "r") as c_file:
+with open("ext/php_aws_crt.h", "r") as c_file:
     for line in c_file:
-        line = re.sub("#define CRT_VERSION .*",
-                      f"#define CRT_VERSION \"{VERSION}\"", line)
+        line = re.sub("#define PHP_AWSCRT_VERSION .*",
+                      f"#define PHP_AWSCRT_VERSION \"{VERSION}\"", line)
```

Comment:
this is replacing `#define` but you're changing to file (below) to be `#    define`

maybe just replace starting at the "define"

Reply:
maybe assert afterwards that `line.contains(VERSION)`?

Reply:
Yeah, it's auto format, but it's really related to the line I was trying to replace. But, we can add the assert for sure.